src/Adapter/LegacyContext.php

```diff
@@ -206,6 +206,28 @@ public function getLanguages($active = true, $id_shop = false, $ids_only = false
         return $languages;
     }
 
+    /**
+     * Returns all languages - active and inactive ones. The first one is the employee default one.
+     *
+     * @return array
+     */
+    public function getLanguagesIncludingInactive()
+    {
+        $languages = Language::getLanguages(false);
+
+        $defaultLanguageFirst = $this->getLanguage();
+        usort($languages, function ($a, $b) use ($defaultLanguageFirst) {
```

Comment:
why employee language has to be first? :thinking:

Reply:
was part of legacy stuff - did not wanted to change that :D

Reply:
It allows all the BO forms to display the default language first.
Then, the additional selectors allow the merchant to set a string for another language.